kml/test_waypoints_to_kml.py

```diff
@@ -0,0 +1,43 @@
+"""
+Test Process.
+"""
+
+import pathlib
+import pytest
+
+from modules import waypoints_to_kml
+
+
+@pytest.fixture
+def waypoints():
+    return [(37.7749, -122.4194), (34.0522, -118.2437), (40.7128, -74.0060)]
+
+
+def test_waypoints_to_kml_with_save_path(waypoints: "list[tuple[float, float]]",
+                                         tmp_path):
+    """
+    Basic test case to save KML to the correct path when provided.
+    """
+    actual_document_name = "actual_kml_document"
+
+    # Build a temporary directory using tmp_path so the KML files are cleaned after the tests are run
+    save_path = pathlib.Path(tmp_path, "path", "to", "save")
```

Comment:
Is path/to/save necessary? Why not just the original Pytest temporary directory?